src/components/screens/Consul/Debates/NewDebate.js

```diff
@@ -107,18 +101,18 @@ export const NewDebate = ({ navigation, data, query }) => {
 
   return (
     <SafeAreaViewFlex>
-      <WrapperWithOrientation>
+      <Wrapper>
         {Inputs.map((item, index) => (
-          <Wrapper key={index} style={styles.noPaddingTop}>
+          <View key={index}>
             <Input {...item} control={control} rules={item.rules} />
-          </Wrapper>
+          </View>
         ))}
 
         <WrapperHorizontal>
           <Checkbox
-            title={text.termsOfServiceLabel}
+            title={texts.consul.startNew.termsOfServiceLabel}
             link={'https://beteiligung.bad-belzig.de/conditions'}
```

Comment:
this link should be placed in texts or elsewhere, instead of having it hardcoded.
in addition the link should be opened in the internal web screen instead of the external browser.

Reply:
I wrote a note. I will update. 👍

Reply:
fa2102e9106ddc1e77dd762050edc3639ac85ac8

Reply:
nice